scripts/build_natura_raster.py

```diff
@@ -64,28 +64,6 @@
 # Requirement to set path to filepath for execution
 # os.chdir(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This file has been modified while you were working on your branch, please, revert it to the one in new main.